src/attr.rs

```diff
@@ -105,15 +159,13 @@ impl<'s> Attributes<'s> {
 
     /// Returns a reference to the value corresponding to the attribute key.
     #[must_use]
-    pub fn get(&self, key: &str) -> Option<&str> {
+    pub fn get(&self, key: &str) -> Option<&Attr<'s>> {
         self.iter().find(|(k, _)| *k == key).map(|(_, v)| v)
     }
```

Comment:
because `Attr` may only contain a single continuous `&str`, it is impossible to represent concatenated classes. `attrs.get("class")` here will only return the first class, it seems.

Reply:
For now, maybe we should use an owned cowstr in those cases?

Reply:
Yes, that's exactly what I was thinking in this https://github.com/hellux/jotdown/pull/19#discussion_r1127039234. I agree, a CowStr seems reasonable for the time being.

Reply:
Using CowStr now.